30DayChallengeHR/Day4.js

```diff
@@ -0,0 +1,64 @@
+process.stdin.resume();
+process.stdin.setEncoding('ascii');
+
+var input_stdin = "";
+var input_stdin_array = "";
+var input_currentline = 0;
+
+process.stdin.on('data', function (data) {
+    input_stdin += data;
+});
+
+process.stdin.on('end', function () {
+    input_stdin_array = input_stdin.split("\n");
+    main();    
+});
+
+function readLine() {
+    return input_stdin_array[input_currentline++];
+}
+
+function Person(initialAge){
+    // Add some more code to run some checks on initialAge
+    this.age = initialAge;
+
+    if (initialAge < 0) {
+        this.age = 0;
+        console.log("Age is not valid, setting age to 0.")
+        initialAge = this.age;
```

Comment:
You don't need to re-assign initialAge to age here, as it's never used again and passed in by value